src/components/StateEligibility.js

```diff
@@ -48,71 +75,44 @@ export default function StateEligibility() {
-                            </a>
-                        </EligibilityGroupItem>
-                        <EligibilityGroupItem>
-                            <a
-                                href="https://www.mass.gov/info-details/covid-19-vaccinations-for-people-ages-75-and-older"
-                                rel="noreferrer"
-                                target="_blank"
-                            >
-                                Individuals age 75 and older
-                            </a>
-                        </EligibilityGroupItem>
-                        <EligibilityGroupItem>
-                            <a
-                                href="https://www.mass.gov/info-details/massachusetts-covid-19-vaccination-phases#phase-1-"
-                                rel="noreferrer"
-                                target="_blank"
-                            >
-                                People in Phase 1 (healthcare, nursing homes,
-                                etc.)
-                            </a>
-                        </EligibilityGroupItem>
-                        {/* TODO - remove the following div after March 11, and update link to be appropriate link from https://www.mass.gov/covid-19-vaccine*/}
-                        <br></br>
-                        <br></br>
-                        <div>Eligible to sign up starting March 11: </div>
-                        <br></br>
-                        <EligibilityGroupItem>
-                            <a
-                                href="https://www.mass.gov/info-details/massachusetts-covid-19-vaccination-phases"
-                                rel="noreferrer"
-                                target="_blank"
-                            >
-                                Pre-K to 12 teachers and staff, early educators,
-                                and child care workers
-                            </a>
-                        </EligibilityGroupItem>
+                        {criteriaGroups.map((g, idx) => {
+                            return (
+                                // https://reactjs.org/docs/reconciliation.html#recursing-on-children
+                                // "The key only has to be unique among its siblings, not globally unique"
+                                // However the interaction with fragments is unclear, so ensure
+                                // title fragments and listitems are unique, as they are siblings.
+                                <React.Fragment key={"title" + idx}>
+                                    {
+                                        // We don't display a title for the first group, for some reason...
+                                        g.title ? (
+                                            <>
+                                                <br />
+                                                <br />
+                                                <div>{g.title}</div>
```

Comment:
nitpick I'm not requiring you fix: instead of relying on the correct number of breaks it'd be cleaner to style the div with padding.

Reply:
This PR attempted to make no visible changes in the final DOM other. Those `<br/>`s were there before and I agree they're probably wrong but someone wiser than I approved them in the past.